releasenotes/notes/cloud_provider_agent_configuration-e722d1992e75ed75.yaml

```diff
@@ -0,0 +1,18 @@
+# Each section from every releasenote are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+features:
+  - |
+    Introducing the cloud_provider option in agent configuration to stop querying cloud provider metadata endpoints that the host will not use.
+enhancements:
+  - |
+    The cloud_provider option allows users to configure their cloud provider in datadog.yaml
+    
```

Comment:
`enhancements` should be use to change on feature already released and `security` for security issues/fix. Here I think we only nee `features`.